cmd/immuclient/command/cmd.go

```diff
@@ -76,3 +77,14 @@ func commandNames(cms []*cobra.Command) []string {
 	}
 	return args
 }
+
+// fprintln is equivalent to fmt.Fprintln but appends new line only if one doesn't exist.
+func fprintln(w io.Writer, msg string) {
```

Comment:
what do you think about printing original input if it ends it a line break and use `fmt.Fprintln` with original input otherwise?

Reply:
yeah, I think I can figure out a better solution for double newline case. Let me look.

Reply:
I have made the returned value consistent everywhere with a newline now. This way, we don't have to do a check and figure out whether to append a newline. Please have a look.